packages/@react-stately/color/src/Color.ts

```diff
@@ -162,6 +169,9 @@ class RGBColor implements ColorValue {
     }
   }
 
+  toInt() {
+    return this.red * RED_MULTIPLIER + this.green * GREEEN_MULTIPLIER + this.blue;
```

Comment:
You could use bit ops for this: 😄 

```suggestion
    return this.red << 16 | this.green << 8 | this.blue;
```

This works because each component is 8 bits (0-255), so you shift each component over by 8 bits and combine them together with bitwise or so they sit next to each other in a single bitstring.

Reply:
nice! might be clearer to read, but may not be more efficient i don't think
to do the bit operators all the numbers have to be converted from 64 bit floating to 32 bit signed first
I don't think anything will be happening fast enough to worry about the speed though